src/HW15/Burger.java

```diff
@@ -0,0 +1,37 @@
+package HW15;
+
+public class Burger {
```

Comment:
U should use inhertance for this task, like buger-> cheseburgeg and and burger -> bigmak burger